### ISSUE TITLE
ESLint - Stylish reporter now reports absolute paths.

### ISSUE THREAD

**Issue by egamma** [Microsoft/vscode#2202]
Extracted from comment from @felixfbecker in issue #1420.

I have the exact same problem with eslint:

![image](https://cloud.githubusercontent.com/assets/10532611/12521660/30167838-c14c-11e5-8eef-b7a5d2004935.png)

tasks.json:
```json
{
  "version": "0.1.0",
  "command": "npm",
  "args": ["--loglevel", "silent", "run"],
  "isShellCommand": true,
  "showOutput": "silent",
  "tasks": [
    {
      "taskName": "lint",
      "problemMatcher": "$eslint-stylish"
    }
  ]
}
```
Setting the problem matcher with fileLocation: absolute works, but imo this should not be needed. I have the latest eslint version installed so $eslint-stylish should set this automatically.

**Comment by dbaeumer** [comment]
The default of the eslint-stylish matcher is fileLocation: "relative" since eslint stylish usually reports relative paths. @felixfbecker: could it be that you configured eslint to report absolute paths.

**Comment by felixfbecker** [comment]
I don't know how and where I could have set that, but eslint has been reporting absolute paths for me as far as I can remember. What I don't get is why you don't just call `path.resolve()` on it. That would work with both.

**Comment by dbaeumer** [comment]
I restested it and interestingly paths are now reported as absolute paths. I digged around a little and the reason is as follows: in the past the eslint-stylish reporter was optional (see https://github.com/sindresorhus/eslint-stylish) and reported relative paths. It is now part of eslint and reports paths absolute. I will keep the issue open to adopt that. 

Regarding your suggestion: because we say output (not eslint) having a leading slash but not being absolute :-). The issue we have open is to have a setting automatic (using path.resolve) which would be the default. But it will not make it for Jan.
